tests/project-list.spec.ts

```diff
@@ -0,0 +1,54 @@
+// modules
+import { expect, Locator, test } from "@playwright/test";
+import { capitalize } from "lodash";
+
+// files
+import projectsMockedRespBody from "./fixtures/projects.json";
+
+test.describe("Projects list", () => {
+  const endpoint = process.env.NEXT_PUBLIC_API_BASE_URL;
+
+  test.beforeEach(async ({ page }) => {
+    // load the base URL
+    await page.goto("http://localhost:3000/dashboard");
+
+    // set the API mock
+    await page.route(`${endpoint}/projects`, async (route) => {
+      await route.fulfill({
+        body: JSON.stringify(projectsMockedRespBody),
+      });
+    });
+  });
+
+  test.describe("'Desktop' resolution", () => {
+    test.beforeEach(async ({ page }) => {
+      await page.setViewportSize({ width: 1025, height: 900 });
+    });
+
+    test("renders the projects", async ({ page }) => {
+      const languageNames = ["Node.js", "React", "Python"];
+
+      // get project cards
+      const projectCards: Locator[] = await page
+        .getByTestId("project-cards-list")
+        .locator("li [data-testID]")
+        .all();
```

Comment:
I'm not sure this is really necessary. I think we're 

1. looking to test the number of cards on the page (useful-ish when testing, say, query limits or pagination)
2. looking to see that each card renders the proper data.

So, we need to total number of cards and then a way to grab each individual card, right?

```javascript
      const projectCards = page.locator("main ul").getByRole("listitem");

      await expect(projectCards).toHaveCount(3);
```

And then we get to take advantage of our test id tags with
```javascript
      for (const projectData of projectsMockedResponseBody.data) {
        const card = page.getByTestId(projectData.id);
...
      }

```